pkg/3scale/amp/operator/backend_options_provider.go

```diff
@@ -156,6 +156,19 @@ func (o *OperatorBackendOptionsProvider) setResourceRequirementsOptions() {
 		o.backendOptions.WorkerResourceRequirements = v1.ResourceRequirements{}
 		o.backendOptions.CronResourceRequirements = v1.ResourceRequirements{}
 	}
+
+	// DeploymentConfig-level ResourceRequirements CR fields have priority over
+	// spec.resourceRequirementsEnabled, overwriting that setting when they are
+	// defined
+	if o.apimanager.Spec.Backend.ListenerSpec.Resources != nil {
```

Comment:
`o.apimanager.Spec.Backend.ListenerSpec` not nil always?

Reply:
Yes.

We initialize it in the `setDefaults` method for the CR